From=20971e75934bc64627226d61565f68aff3c9b0000b Mon Sep 17 00:00:00 2001
From: Rex Dieter <rdieter@math.unl.edu>
Date: Mon, 12 Sep 2016 11:58:12 -0500
Subject: [PATCH] fix FTBFS against cups-2.2

cupsGetPPD2 moved to cups/ppd.h

BUG: 366483
FIXED-IN: 16.08.2
---
 libkcups/KCupsRequest.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libkcups/KCupsRequest.cpp b/libkcups/KCupsRequest.cpp
index 39a5cf5..3625bec 100644
--- a/libkcups/KCupsRequest.cpp
+++ b/libkcups/KCupsRequest.cpp
@@ -29,6 +29,7 @@
 #include <QStringBuilder>

 #include <cups/adminutil.h>
+#include <cups/ppd.h>

 #define CUPS_DATADIR    "/usr/share/cups"
